### PR TITLE
Update linter and formatter to match

### PR DESCRIPTION
**Pull request by sethvargo**
Match linter and esrules

Do you want me to also update husky in this PR?

### CODE REVIEW

**Comment by craigdbarber** [comment]
> Match linter and esrules
> 
> Do you want me to also update husky in this PR?

Sure, that'd be great. Thanks!

File: setup-gcloud/tests/download-util.test.ts [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+describe('#downloadAndExtractTool', function() {
+  beforeEach(async function() {
+    await io.rmRF(toolDir);
+    await io.rmRF(tempDir);
+  });
+
+  afterEach(async function() {
+    await io.rmRF(toolDir);
+    await io.rmRF(tempDir);
+  });
```
Comment (craigdbarber) [inline setup-gcloud/tests/download-util.test.ts]: Please be sure to preserve the original test timeouts, here and throughout. The MacOS runners can be very flaky if you don't

**Comment by craigdbarber** [comment]
Thanks for the contribution! One general note, using git mv is preferred over git rm and git add, as it preserves the history for the file.

File: setup-gcloud/tests/download-util.test.ts (hunk 1 above)
Comment (sethvargo) [reply setup-gcloud/tests/download-util.test.ts]: Each test has a an individual timeout of 30s per the mocha command in the package.json. Do you want it to be higher?

**Comment by sethvargo** [comment]
@craigdbarber ack, sorry

File: setup-gcloud/tests/download-util.test.ts (hunk 1 above)
Comment (craigdbarber) [reply setup-gcloud/tests/download-util.test.ts]: 30s works. Thanks for clarifying :)

**Pull request merged**